eldritch/test_mythos.py

```diff
@@ -1470,6 +1471,85 @@ def setUp(self):
       monster.idx = idx
       monster.place = self.state.monster_cup
 
+  def testActivateRumor13(self):
+    self.state.mythos.append(Mythos13())
+    self.state.event_stack.append(Mythos(self.char))
+    self.resolve_until_done()
+    self.assertTrue(self.state.rumor)
+    self.assertEqual(self.state.rumor.name, "Mythos13")
+    self.assertFalse(self.state.rumor.failed)
+    self.assertNotIn(self.state.rumor, self.state.mythos)
+
+  def testRumor13Progress(self):
+    self.state.event_stack.append(IncreaseTerror(6))
```

Comment:
Add some allies to the deck before progressing the rumor and make sure the allies go back to the box? And also that the rumor still progresses even if the allies deck is empty?